src/rest/rest_web_server.cpp

```diff
@@ -139,7 +147,7 @@ void RestWebServer::InitializeListenFd(void)
     mListenFd = SocketWithCloseExec(AF_INET, SOCK_STREAM, 0, kSocketNonBlock);
     VerifyOrExit(mListenFd != -1, err = errno, error = OTBR_ERROR_REST, errorMessage = "socket");
 
-    ret = setsockopt(mListenFd, SOL_SOCKET, SO_REUSEADDR, reinterpret_cast<char *>(&optval), sizeof(optval));
+    ret = setsockopt(mListenFd, SOL_SOCKET, SO_REUSEADDR | SO_REUSEPORT, reinterpret_cast<char *>(&optval), sizeof(optval));
```

Comment:
Why is `SO_REUSEPORT` necessary?

Reply:
I was just trying different options to see what happens.